src/sentry/api/utils.py

```diff
@@ -346,3 +366,68 @@ def get_auth_api_token_type(auth: object) -> str | None:
     if is_api_key_auth(auth):
         return "api_key"
     return None
+
+
+# NOTE: This duplicates OrganizationEventsEndpointBase.handle_query_errors
+# TODO: move other references over to this implementation rather than the organization_events implementation
+@contextmanager
+def handle_query_errors(self) -> Generator[None, None, None]:
+    try:
+        yield
+    except InvalidSearchQuery as error:
+        message = str(error)
+        # Special case the project message since it has so many variants so tagging is messy otherwise
+        if message.endswith("do not exist or are not actively selected."):
```

Comment:
should this be a const value so we run into less issues with string comparisons? Can we use the constant where the exception is thrown?
Just looking at the error, do we know not the difference between the 2? Would help the user better understand what the state of things are for the issue (especially if they contact us for support haha)

Reply:
Fair question - TBH, not entirely sure, but this is all just copy/pasta from (here)[https://github.com/getsentry/sentry/blob/master/src/sentry/api/bases/organization_events.py#L210] so i'm not looking to change too much